pkg/services/accesscontrol/roles.go

```diff
@@ -37,8 +40,9 @@ var usersOrgReadRole = RoleDTO{
 }
 
 var usersOrgEditRole = RoleDTO{
-	Name:    usersOrgEdit,
-	Version: 1,
+	Name:        usersOrgEdit,
+	Version:     2,
+	Description: "Allows every read action for user organizations and in addition allows to administer user organizations.",
```

Comment:
```suggestion
	Description: "Allows every read action for user organizations and in addition allows to administer user organizations.",
```
Allows [what? to? for?] every read action ... allows [what? / who?] to administer user organizations.

Reply:
Again, I'm trying to propose something, but I like the "administer" word being large enough to encompass future changes:
```suggestion
	Description: "Role to allow users everything usersOrgReadRole allows and to add/update/remove users from an organization.",
```